packages/ssz/src/viewDU/listComposite.ts

```diff
@@ -66,6 +66,47 @@ export class ListCompositeTreeViewDU<
     return this.type.getViewDU(newRootNode) as this;
   }
 
+  /**
+   * Returns a new ListCompositeTreeViewDU instance with the values from `index` to the end of list
+   *
+   * ```ts
+   * const nodes = getChunkNodes()
+   * return listFromChunkNodes(node.slice(index))
+   * ```
+   *
+   * Note: If index === n, returns an empty list of length 0
+   *
+   */
+  sliceFrom(index: number): this {
```

Comment:
can you add a sliceFrom method for ListBasicTreeViewDU ? Or just make an issue for it?

Reply:
> can you add a sliceFrom method for ListBasicTreeViewDU ? Or just make an issue for it?

Is it for the sake of completion? There is no use of `sliceFrom` for `ListBasicTreeViewDU` afaik.